.circleci/config.yml

```diff
@@ -156,6 +156,7 @@ jobs:
           sudo chown -R circleci /go
           mkdir -p /home/circleci/logs
       - checkout
+      - run: make git/master
```

Comment:
please don't put `/` in a (.phony) make target 

how is it different from "make pull" that's already there ? try that one ? (you can remove submodule-sync if you do make pull)

in which case/why would the just checked out source tree need a pull ?

Reply:
Re. "/" - why not ? OpenWRT uses this convention (package/foo), I don't like inventing new delimiters and the targets need to be organized a bit eventually. Replaced it with a . since we use it in few targets to avoid another bikesheding, but like / more.

Added comments - it's what you asked for, pulling the changes from master. Submodule-sync seems to sync vendor - not sure what the git pull does there, is redundant with checkout in CI (as you mentioned, we just checked out), but I assumed it's for dev using make ? I didn't put it there, the new one is pulling from master into the branch.

Reply:
`x/y` is a path for instance you would `make bin/foo` if you're target is not a real target/file, don't use a / it's just confusing. thanks for the change

yes the `make pull` I added for dev workflow to replace `git pull` and take care of the submodule part, so you're saying this new target is for CI to merge master on the PR?
maybe should be --ff-only ? somehow prow uses 'merge' but I don't know/whichever works

will it abort the build in a way that's easy to understand if the PR has conflicts ?